src/valuetype_idtype.ts

```diff
@@ -105,11 +105,20 @@ function parseIDType(def: ITypeDefinition, data: any[], accessor: (row: any, val
   return [];
 }
 
+function getMarkup(this: ValueTypeEditor, current: ValueTypeEditor, def: ITypeDefinition): string {
+  const allIDTypes = listidtypes().filter((idType) => !isInternalIDType(idType));
+
+  return `<optgroup label="Identifier" data-type="idType">
+        ${allIDTypes.map((type) => `<option value=${type.name} ${current && current.id === this.id && type.name === def.idType ? 'selected="selected"' : ''}>${type}</option>`).join('\n')}
```

Comment:
why is the value the `name` and not the `id` and the label of the option the toString instead of the `name`